chars/src/main.rs

```diff
@@ -4,10 +4,15 @@ use chars::display;
 use chars::human_names;
 
 fn main() {
-    let args = env::args()
-        .skip(1)
-        .flat_map(|argument| human_names::from_arg(argument.as_ref()));
-    for c in args {
-        display::describe(c);
+    let args = env::args().skip(1);
+    for argument in args {
+        let results = human_names::from_arg(argument.as_ref());
+        if results.len() == 0 {
+            eprintln!("No results for “{}”.", argument);
```

Comment:
Hm, so this will print `no results for timey-wimey` and `no results for stuff` when you run `chars timey-wimey stuff`. This seems reasonable and right for the CLI interface we have at the moment, but I think by now I would also like to restructure the interface, especially for things like #81.

Not a blocker to make this go in, but a good reminder that the CLI at the moment isn't the best.